src/util.cpp

```diff
@@ -6686,7 +6686,8 @@ bool isURL(const QCString &url)
 {
   QCString loc_url = url.stripWhiteSpace();
   return loc_url.left(5)=="http:" || loc_url.left(6)=="https:" ||
-         loc_url.left(4)=="ftp:"  || loc_url.left(5)=="file:";
+         loc_url.left(4)=="ftp:"  || loc_url.left(5)=="ftps:"  || loc_url.left(5)=="sftp:"  ||
+         loc_url.left(5)=="file:";
```

Comment:
I was wondering if for consistency with the `URLPROTOCOL` pattern in `doctokenizer.l`, the prefixes `news:`, `irc:`, and `ircs`: should be added here as well?

Reply:
Probably both sets should have the same list.